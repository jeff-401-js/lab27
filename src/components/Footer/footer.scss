@import '../../app.scss';

footer {
  max-width: 150px;
  height: 10px;
  margin-top: 20px;
  font-size: 10px;
  color: rgb(219, 151, 24);
  font-family: $mainFont;
  background-color: rgb(33, 12, 151);
}