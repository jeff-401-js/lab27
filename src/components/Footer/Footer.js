import React from 'react';
import './footer.scss';

/**
 * Footer class which simply contains the footer information
 */
class Footer extends React.Component {
  render() {
    return <footer>&copy; 2018 Code Fellows</footer>;
  }
}

export default Footer;
