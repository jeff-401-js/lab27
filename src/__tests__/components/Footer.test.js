import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../../components/Footer/Footer';

describe('<Counter/> Snapshot Test', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Footer/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
