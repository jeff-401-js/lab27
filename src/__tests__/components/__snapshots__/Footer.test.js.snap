// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Counter/> Snapshot Test renders correctly 1`] = `
<footer>
  Â© 2018 Code Fellows
</footer>
`;
